### PR TITLE
Docsp-20447 mongosh 1 1 9

### PR DESCRIPTION
**Pull request by ianf-mongodb**
Jira:
https://jira.mongodb.org/browse/DOCSP-20447

Build:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=61e7174ebcf82347e52ebdc2

Staging:
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-20447-mongosh-1-1-9/changelog/

### CODE REVIEW

File: source/changelog.txt [hunk 1]
```diff
@@ -12,6 +12,23 @@
...
    :class: singlecol
 
+v1.1.9
+------
+
+*Released January 18, 2022*
+
+New features in this release:
+
+- :issue:`MONGOSH-1015` – mongosh no longer overrides appName if it
```
Comment (jeff-allen-mongo) [inline source/changelog.txt:22]:
```suggestion
- :issue:`MONGOSH-1015` – ``mongosh`` no longer overrides ``appName`` if it 
```

File: source/changelog.txt [hunk 2]
```diff
@@ -12,6 +12,23 @@
...
+*Released January 18, 2022*
+
+New features in this release:
+
+- :issue:`MONGOSH-1015` – mongosh no longer overrides appName if it 
+  was present in the connection string.
+
+- :issue:`MONGOSH-1073` – You can now pass BSON number objects to the 
+  legacy BSON number constructors, for example:
+  ``NumberInt(NumberInt(n))`` now works like it did in the legacy shell.
```
Comment (jeff-allen-mongo) [inline source/changelog.txt:27]: I think this should be separate sentences.

```suggestion
  legacy BSON number constructors. For example,
  ``NumberInt(NumberInt(n))`` now works like it did in the legacy shell.
```

**Review by jeff-allen-mongo — COMMENTED** [review]
@ianf-mongodb , thanks for handling these notes. Just a couple minor cleanup comments.

**Comment by ianf-mongodb** [comment]
This should be gtg @jeff-allen-mongo

**Review by jeff-allen-mongo — APPROVED** [review]

**Pull request merged**